packages/vite-plugin-shopify/src/html.ts

```diff
@@ -21,6 +22,13 @@ export default function shopifyHTML (options: ResolvedVitePluginShopifyOptions):
     configResolved (resolvedConfig) {
       // Store reference to resolved config
       config = resolvedConfig
+
+      // Check for alias generated by vite-plugin-shopify-modules
+      const modulesAlias = config.resolve.alias.find((value) => value.find === '@modules')
```

Comment:
@ScottPolhemus Looking great on my end

What if we assumed that `additionalEntrypoints` are relative to the `themeRoot` so that it can support any path?

1. In the the plugin, we could pass in any path to load entries
```
plugins: [shopify({
  additionalEntrypoints: ['foo/**/*.js']
})]
```

2. In liquid files, we could load any entry given its path
```
{% render 'vite-tag' with 'foo/bar.js' %}
```

3. Then, vite-tag.liquid would use that path to output the condition and load the entry
```
# development
{% if vite-tag == 'foo/bar.js' %}
<script src="http://localhost:5173/foo/bar.js" type="module"></script>
{% endif %}

# production
{% if vite-tag == 'foo/bar.js' %}
 <script src="{{ 'bar.js' | asset_url }}" type="module" crossorigin="anonymous"></script>
{% endif %}
```

Reply:
Hey guys, Implemented this idea in #7 please let me know what you think